tests/integration/operator/switch_cr_test.go

```diff
@@ -123,7 +123,36 @@ func TestController(t *testing.T) {
 			cleanupInClusterCRs(t, cs)
 
 			// test operator remove command
-			scopes.Framework.Infof("checking operator remove command")
+			scopes.Framework.Infof("checking operator remove command for default revision")
```

Comment:
I don't think this's the right place, you should create a new test case.

Reply:
OK, I will modify it later